### PR TITLE
Add i18n localized strings to images component

### PR DESCRIPTION
**Pull request by rak-phillip**
This addresses a small piece of tech debt by adding i18n localized strings to the images component. 

#585

### CODE REVIEW

**Review by mook-as — APPROVED** [review]
Yay!

We'll need to figure out how to manage resources copied from dashboard though; previously, we could just replace the translations files wholesale with the upstream copy.  Now we'll need to figure out something for merging.

**Comment by rak-phillip** [comment]
> Yay!
> 
> We'll need to figure out how to manage resources copied from dashboard though; previously, we could just replace the translations files wholesale with the upstream copy. Now we'll need to figure out something for merging.

We might want to look at strategies for keeping the two separate. I've created #589 so we can track this progress

File: src/components/Images.vue [hunk 1]
```diff
@@ -25,44 +25,44 @@
       <Card :show-highlight-border="false" :show-actions="false">
         <template #title>
           <div class="type-title">
-            <h3>Image Acquisition</h3>
+            <h3>{{ t('images.manager.title') }}</h3>
```
[the remark this answers is not in the document]
Comment (rak-phillip) [reply src/components/Images.vue:28]: It really comes down to style preference. I think we prefer the mustache syntax in dashboard, but I'm willing to stick with whatever we prefer for Rancher Desktop.

[the remark this answers is not in the document]
Comment (mook-as) [reply src/components/Images.vue:28]: Oh I have no idea what we prefer; just went through Vue docs at one point… :D

**Pull request merged**